EMIP/1-1000/EMIP00937.xml

```diff
@@ -30,16 +31,20 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                   <idno facs="EMIP/Codices/937/" n="130">Mekele Mikael 160</idno>
                </msIdentifier>
                <msContents>
-                  <summary>Hosea (1r-6v), Amos (6v-10v), Micah (10v-14r), Joel (14r-16r), Obadiah (16rv), Jonah (16v-17v), Nahum (17v-19r), Habakkuk (19r-20v), Zephaniah (20v-22r), Haggai (22r-23r), Zechariah (23r-29r), Malachi (29r-31r), ? (31rv), Jeremiah (32r-84v), Henoch with marginal commentary (85r-120r), ? (120r)</summary>
+                  <summary>Hosea (1r-6v), Amos (6v-10v), Micah (10v-14r), Joel (14r-16r), Obadiah
+                     (16rv), Jonah (16v-17v), Nahum (17v-19r), Habakkuk (19r-20v), Zephaniah
+                     (20v-22r), Haggai (22r-23r), Zechariah (23r-29r), Malachi (29r-31r), ? (31rv),
+                     Jeremiah (32r-84v), Henoch with marginal commentary (85r-120r), ?
+                     (120r)</summary>
                   <msItem xml:id="ms_i1">
-                     <textLang mainLang="gez">Gǝ‘ǝz</textLang>
+                     <locus from="85r" to="120r"/>
+                     <title ref="LIT1340EnochE">Henok</title>
```

Comment:
I have not seen this one.

This cannot be ms_i1. 

i1 are Prophets
then you have i1.1 Hosea, i1.2 Amos etc.

or no ms_i at all

Thank you